typescript/scripts/README.adoc

```diff
@@ -0,0 +1,38 @@
+:toc: macro
+
+= tBTC v2 Refund script
+
+This script explains what to do in case you need to refund your BTC.
+
+toc::[]
+
+== Refund script
+
+=== Prerequisites
+
+Please make sure you have the following prerequisites installed on your machine:
+
+- https://nodejs.org[Node.js] >14.18.0
+- https://yarnpkg.com[Yarn] >1.22.19
+
+tBTCv2 system prerequisite is that you can refund your BTC only after `refundLocktime` expires, which is currently set to 9 month.
+
+=== Running a refund script
+
+Step 1. Clone https://github.com/keep-network/tbtc-v2[tbtc-v2] repository to your local disc. `git clone https://github.com/keep-network/tbtc-v2`
+
+Step 2. Navigate to a root directory and run the following command with your arguments:
+```
+./typescript/scripts/refund.sh --deposit-json-path <deposit-JSON-file-path> --amount <amount> --transaction-id <transaction-id> --private-key <wallet-private-key> --transaction-fee <transaction-fee> --electrum-host <electrum-host> --electrum-port <electrum-port> --electrum-protocol <electrum-protocol>
+```
+Every argument must be provided but electrum related info. You can set you own electrum connection if for some reason the default one doesn't work. However, the following arguments must be provided:
+
+- `deposit-JSON-file-path` Path to a file that you downloaded during the BTC deposit in the Threshold dApp.
+- `amount` Exact BTC amount in satoshi that was deposited. It will not process a partial refund.
+- `transaction-id` A transaction ID of your BTC deposit.
```

Comment:
We will also have something like deposit-transaction-index here.
- `deposit-transaction-index` The position of your BTC deposit output in the deposit transaction. This can be checked using block explorer. If it's first on the list then the index is `0`, if it's second then the index is `1`, etc.

Reply:
Yes, added.